helsinki_theme/templates/helsinki_theme/base.html

```diff
@@ -19,7 +19,11 @@
     <div class="col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3">
       <div class="content-card">
         <header>
-          <img src="{% static "hel-bootstrap-3/src/assets/helsinki-logo-black.svg" %}" class="logo" alt="Helsinki Helsingfors">
+          {% if request.LANGUAGE_CODE == "sv" %}
```

Comment:
Could maybe use the `get_current_language` template tag here?

Reply:
Changed. How does it look?